index.bs

```diff
@@ -568,6 +568,8 @@ ReadableStream(<var>underlyingSource</var> = {}, { <var>size</var>, <var>highWat
 </div>
 
 <emu-alg>
+  1. If _writable_ is *undefined* or _readable_ is *undefined*, throw a *TypeError* exception indicating that undefined
+     values are not permitted.
```

Comment:
_readable_ does not exist (use **this**). But that part is redundant with the next line, which will throw anyway. Implementations are free to do an identity check against undefined and throw a different TypeError if they want, since that's unobservable.

Will fix and merge.